components/Tabs2/tests/unit/MDCTabBarViewTests.m

```diff
@@ -19,10 +19,103 @@
 @interface MDCTabBarViewTests : XCTestCase
 @end
 
-@implementation MDCTabBarViewTests
+@implementation MDCTabBarViewTests {
+  MDCTabBarView *_tabBarView;
+  UITabBarItem *_itemA;
+  UITabBarItem *_itemB;
+  UITabBarItem *_itemC;
+}
+
+- (void)setUp {
+  [super setUp];
+
+  _tabBarView = [[MDCTabBarView alloc] init];
+  _itemA = [[UITabBarItem alloc] initWithTitle:@"A" image:nil tag:0];
+  _itemB = [[UITabBarItem alloc] initWithTitle:@"B" image:nil tag:0];
+  _itemC = [[UITabBarItem alloc] initWithTitle:@"C" image:nil tag:0];
+}
+
+- (void)tearDown {
+  _itemA = nil;
+  _itemB = nil;
+  _itemC = nil;
+  _tabBarView = nil;
+
+  [super tearDown];
+}
 
 - (void)testInitCreatesObject {
   XCTAssertNotNil([[MDCTabBarView alloc] init]);
 }
 
+/// Tab bars should by default select the first item in their items array.
+- (void)testSelectsFirstItemByDefault {
+  _tabBarView.items = @[ _itemA, _itemB, _itemC ];
```

Comment:
Please use `Given`, `When`, and `Then` comments and test structure.  There are [many good resources discussing Given-When-Then testing](http://www.draconianoverlord.com/2017/11/28/given-when-then.html) available on the web, and our code base uses it extensively.

Reply:
Updated.